### ISSUE TITLE
Strider: Strider reports tests successful if skipped via missing "Ready for CI build" label

### ISSUE THREAD

**Issue by gebart** [RIOT-OS/RIOT#3387]
... as long as the static tests are OK.

```

$ bash -e -x -c 'export SHELL=/bin/bash export CI_PULL_REQUEST=3380 ./dist/tools/drone-scripts/build_and_test.sh' custom 55.6s

+ export SHELL=/bin/bash
+ SHELL=/bin/bash
+ export CI_PULL_REQUEST=3380
+ CI_PULL_REQUEST=3380
+ ./dist/tools/drone-scripts/build_and_test.sh
PR not ready for CI build. Only static-tests will be executed!
Begin group static-tests
First, rewinding head to replay your work on top of it...
Applying: boards/Makefile.include.cortexm_common: Add TOOLCHAIN=gcc as an alias for TOOLCHAIN=gnu
Result (static-tests): 0
Summary of test groups run:
static-tests: 0
Build and test: All successful

```

**Comment by gebart** [comment]
added the major label because of the severity (false green lights for tests) even though it is probably an easy fix.

**Comment by gebart** [comment]
Closed via #3409
